public class myclass
{
	public static void main(String args[])
	{
	 	try
	 	{
		int []num={1,2,3};
		System.out.println(num[10]);
		}
		catch(Exception e)
		{
			System.out.println("error occured in the program");
		}
		finally
		{
			System.out.println("try and catch executed successfully");
			}
		
	}
	}
